src/js/components/DataTable/__tests__/DataTable-test.js

```diff
@@ -408,6 +408,64 @@ describe('DataTable', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('rowDetails', () => {
+    const { container, getByText } = render(
+      <Grommet>
+        <DataTable
+          columns={[
+            { property: 'a', header: 'A' },
+            { property: 'b', header: 'B' },
+          ]}
+          data={[
+            { a: 'one', b: 1.1 },
+            { a: 'one', b: 1.2 },
+            { a: 'two', b: 2.1 },
+            { a: 'two', b: 2.2 },
+          ]}
+          rowDetails={row => <Box>{row.a}</Box>}
+          primaryKey="b"
+        />
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+    const headerCell = getByText('A');
```

Comment:
This will click in the table header cell. I think we need to click the expand/collapse control in the row.

Reply:
Addressed the change.